product_docs/docs/biganimal/release/using_cluster/04_backup_and_restore.mdx

```diff
@@ -36,7 +36,7 @@ BigAnimal retains backups for 30 days by default. You can restore backups into a
 
 The restore operation is available for any cluster that has at least one available backup. For newly created clusters, the initial backup becomes available a few minutes after the new cluster is fully operational.
 
-#### Single-node or high-availability clusters restore
+#### Single-node or primary/standby high-availability clusters restore
```

Comment:
Can we change this and the next one to Restoring xxx clusters?

Reply:
might be easier to merge this PR and then fix the headings in this topic. I've made some major changes to it in another [PR](https://github.com/EnterpriseDB/docs/pull/4584).

Reply:
done